everyvoice/cli.py

```diff
@@ -587,6 +591,7 @@ def test(suite: TestSuites = typer.Argument("dev")):
 
 
 @app.command()
+@merge_args(inference_base_command_interface)
```

Comment:
Hm, right, I guess it's fine to have the inference base command also add to the demo. nice.